conda/base/context.py

```diff
@@ -478,7 +480,12 @@ class Context(Configuration):
 
     no_plugins = ParameterLoader(PrimitiveParameter(NO_PLUGINS))
 
-    def __init__(self, search_path=None, argparse_args=None, **kwargs):
+    def __init__(
+        self,
+        search_path=None,
+        argparse_args=None,
+        **kwargs,
+    ):
```

Comment:
Since theres no long a change here we should avoid reformatting

```suggestion
    def __init__(self, search_path=None, argparse_args=None, **kwargs):
```